package.json

```diff
@@ -11,6 +11,7 @@
     "clean": "rm -fr build",
     "lint": "eslint --fix --ext .ts output/*.ts",
     "build": "npm run clean && tsc",
+    "prepare": "npm run build",
```

Comment:
I think this is redundant, it's the same as build

Reply:
it is not
https://docs.npmjs.com/misc/scripts#description
This is needed so when you simply call npm install it will build the packages correctly, otherwise the build is not run. This is especially visible when you use the package using github path.